source/PluginDev/Assets/GooglePlayGames/BasicApi/PlayGamesClientConfiguration.cs

```diff
@@ -180,7 +204,7 @@ public Builder EnableSavedGames()
                 mEnableSaveGames = true;
                 return this;
             }
-
+            
```

Comment:
minor issue with spaces on blank lines in a couple places.  Please consider cleaning them up, but it won't hold up the merge.